tests/workflows/background_emitter/workflow.py

```diff
@@ -10,9 +10,12 @@
 class ExpensiveEmitter(BaseWorkflowEmitter):
     delay = 0.02
 
-    def emit_event(self, event: WorkflowEvent) -> None:
+    def _emit(self, event: WorkflowEvent) -> None:
         time.sleep(self.delay)
 
+    def emit_event(self, event: WorkflowEvent) -> None:
+        self._emit(event)
```

Comment:
[nit] Let's keep the `time.sleep(self.delay)` here - the self._emit should be empty so that we could assert mock call in our test